dvc/command/diff.py

```diff
@@ -134,17 +133,17 @@ def add_parser(subparsers, parent_parser):
     diff_parser.add_argument(
         "a_ref",
         help=(
-            "Git reference to the old version that you want to compare"
-            " (defaults to HEAD)"
+            "Git reference to the older version to compare "
+            "(defaults to `HEAD`)"
```

Comment:
I'd say that there's no need for ` since it is not a command nor a function

Reply:
Good point. Updated